common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -109,7 +108,7 @@
                 this.sidebar_padding = 10;
                 this.boardName = null;
                 this.current_search = '';
-                this.mode = 'all';
+                this.mode = options.mode || 'commentables';
```

Comment:
I just discovered more bugs in Profile page, cough, this mode is messy!!. Taking a look now.